error-prone-contrib/src/main/java/tech/picnic/errorprone/refasterrules/ReactorRules.java

```diff
@@ -559,7 +559,7 @@ Flux<S> after(Flux<T> flux) {
   static final class MonoFlux<T> {
     @BeforeTemplate
     Flux<T> before(Mono<T> mono) {
-      return Flux.concat(mono);
+      return Refaster.anyOf(Flux.concat(mono), mono.flatMapMany(Flux::just));
```

Comment:
We usually list `Mono`s before `Flux`es, so will change the order :).

Reply:
Noted for the future. :memo: